themes/finna2/templates/RecordDriver/DefaultRecord/data-relatedPublications.phtml

```diff
@@ -3,7 +3,14 @@
   <div class="truncate-field">
     <?php foreach ($data as $publication): ?>
       <div class="related-publications">
-        <?=$this->escapeHtml($publication['title'])?><?php if (!empty($publication['label'])): ?> (<?=$this->escapeHtml($publication['label'])?>)<?php endif; ?>
+        <?php if (!empty($publication['url'])): ?>
+          <a href="<?=$this->escapeHtmlAttr($publication['url'])?>" target="_blank" aria-label="<?=$this->escapeHtmlAttr($publication['title'])?> (<?=$this->transEscAttr('external_link')?>; <?=$this->transEscAttr('Open in a new window')?>)">
+            <i class="fa fa-external-link"></i> <?=$this->escapeHtml($publication['title'])?>
```

Comment:
Käytetään uusissa kohdissa icon-helperiä (tyyliin `<?=$this->icon('external-link')?>`)